concepts/comparisons/introduction.md

```diff
@@ -1,2 +1,53 @@
-#TODO: Add introduction for this concept.
+# Introduction
 
+A comparison operator in python, also called python relational operator, compares the values of two operands and returns `True` or `False` based on whether the condition is met. The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, `"!="`, `"is"`
+
+```python
+>>> 7 > 5
+True
+>>> 99 < 100
+True
+>>> 4 < 4
+False
+>>> 4 <= 4
+True
+>>> 1 >= 1
+True
+>>> 5 == 5
+True
+>>> 6 != 6 # not equal to
+False
+>>> 5 != 6
+True
+```
+
+## Comparison Chaining
+
+Comparisons can be chained arbitrarily, e.g., `x < y <= z` is equivalent to `x < y` `and` `y <= z`, except that `y` is evaluated only once (but in both cases `z` is _not_ evaluated at all when `x < y` is found to be `False`).

```

Comment:
Besides omitting the code examples, I think I would make the same edits to the prose here as I did in the `about.md`.

Reply:
Okay sounds good. 

Now

- code blocks from `introduction.md` have been removed
- prose is matched as per `about.md`